pkgs/applications/editors/neovim/wrapper.nix

```diff
@@ -119,22 +164,17 @@ let
           echo -e "\nGenerating rplugin.vim failed!"
```

Comment:
The double-wrapping can't be removed this easily, because the previous code actually calls `nvim` to generate `rplugin.vim` after the first but before the second wrapping. And the second wrapping then wraps it again with that very `rplugin.vim` that was generated.

What could be done to not double-wrap is to first wrap it with the first set of computed arguments, call it for generating the `rplugin.vim`, then wrap the completely unwrapped neovim again with the same set of computed arguments plus the additional two ones for `rplugin.vim` and the `-u`.

Reply:
:+1:. Please tell me if that works now.